presets/4.3/rates/illusionfpv_freestyle.txt

```diff
@@ -0,0 +1,18 @@
+#$ TITLE: freestyle rates by IllusionFpv
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: OFFICIAL
+#$ KEYWORDS: freestyle rates IllusionFpv quick
```

Comment:
key words should be comma-separated, please

Reply:
added some commas